src/components/ProjectListItemCard/ProjectListItemCard.stories.tsx

```diff
@@ -0,0 +1,27 @@
+import { number, select, text } from '@storybook/addon-knobs';
+import React from 'react';
+import { ProjectStatus } from '../../api';
+import { ProjectListItemCard } from './ProjectListItemCard';
+
+export default { title: 'Components/ProjectListItemCard' };
+
+const statusOptions: string[] & ProjectStatus[] = [
```

Comment:
This should be fine...?
```suggestion
const statusOptions: ProjectStatus[] = [
```